src/coreclr/tools/Common/TypeSystem/Common/FieldDesc.cs

```diff
@@ -45,6 +45,9 @@ public virtual string Name
             get;
         }
 
+        // Get the embedded signature data used to hold custommodifiers and such within a field signature
```

Comment:
Nit / typo - it seems to me it would be a tad more readable if we used a blank to turn these into normal words *custom modifiers*, after all it's not a direct reference to a type name or some such.